### ISSUE TITLE
Unhandled exception: [bug]

### ISSUE THREAD

**Issue by rafalfus** [remyroy/CDDA-Game-Launcher#247]
* Description: [I tried to update the game, and it prompted me if I wanted to delete a file. I said no, and it gave me this]
* Version: 1.3.16
* OS: Windows-7-6.1.7601-SP1 (64-bit)
* Type: `<class 'pywintypes.com_error'>`
* Value: (-2144927744, 'OLE error 0x80270000', None, None)
* Traceback:
```
  File "cddagl\ui.py", line 2437, in timeout
  File "cddagl\ui.py", line 173, in delete_path
  File "site-packages\winutils.py", line 77, in delete
  File "site-packages\winutils.py", line 40, in _file_operation

```

**Comment by remyroy** [comment]
Consider saying yes on that prompt.

The launcher moves the previous version of the game in the recycle bin before updating (so it can keep the current version of the game to restore it back if needed). If there is no more space on your disk, you might need to make room for it on your disk. The previous version of the game is stored in the previous_version sub-directory in the game directory.

**Comment by remyroy** [comment]
It should  be permanently fixed with fdf9488b9f18fe7322a66b22dfefc48dee048648 . It will be part of the next release.
